### PR TITLE
Update media-quality-and-network-connectivity-performance.md

### PR DESCRIPTION
**Pull request by vmahenv**
spelling

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbOnline/optimizing-your-network/media-quality-and-network-connectivity-performance.md [hunk 1]
```diff
@@ -283,7 +283,7 @@ If you have decided that ExpressRoute based on recommendations above is for you,
...
+The following graphic shows an example of this type of deployment. For this example, Contoso is a university located in Orlando, FL. Contoso has 10,000 faculty members and students. The Internet tests from their location to Microsoft edge sites showed greater than 5% packet loss during peak class hours. They have decided to get a dedicated connection to Office 365 using ExpressRoute with over-provisioned bandwidth so they can avoid the network congestion for Office 365 especially for Skype for Business Online Real-Time traffic. They connect to the Microsoft cloud through ExpressRoute at the Atlanta, GA MeetMe site.
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/optimizing-your-network/media-quality-and-network-connectivity-performance.md]:
```suggestion
The following graphic shows an example of this type of deployment. For this example, Contoso is a university located in Orlando, FL. Contoso has 10,000 faculty members and students. The Internet tests from their location to Microsoft Edge sites showed greater than 5% packet loss during peak class hours. They have decided to get a dedicated connection to Office 365 using ExpressRoute with over-provisioned bandwidth so they can avoid the network congestion for Office 365, especially for Skype for Business Online Real-Time traffic. They connect to the Microsoft cloud through ExpressRoute at the Atlanta, GA MeetMe site.
```

**Comment by scanum** [comment]
@tonysmit  PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tonysmit** [comment]
Done.

**Pull request merged**